### ISSUE TITLE
s22-6pm-4 Add field degradation rate to fixtures, form components, table components

### ISSUE THREAD

**Issue by adrianlindell** [ucsb-cs156-s22/s22-6pm-happycows#24]
- [x] There is a field in the fixtures for degradation rate as a decimal number treated as a percentage EDIT: NO LONGER NEEDED
- [x] There is a field in the form components for degradation rate as a decimal number treated as a percentage 
- [x] There is a field in the table components for degradation rate as a decimal number treated as a percentage
- [x] There is a field called "degradation rate" on the create commons, edit commons, and commons list page
- [x] For now, if the API doesn't return this, and your code depends on it, you can add code that adds a fake value of 0.1 into the data returned by the backend; once the API is ready, you can back this code out.
- [x] For acceptance, the actual API values should be used instead of the fake values
